editor/src/templates/editor-canvas.tsx

```diff
@@ -328,7 +293,7 @@ function on(
       return []
     }
   } else if (canvas.keysPressed['z']) {
-    if (event.event === 'MOUSE_UP' && !isDragging(canvas.editorState)) {
+    if (event.event === 'MOUSE_UP') {
```

Comment:
Oh interesting, it seems we've had a bug here for a while then, because this means you can zoom in and out accidentally whilst dragging to move or resize an element. I'd say it's an extremely minor bug tbh, and such an edge case that I'm not surprised we hadn't caught it before.